app.js

```diff
@@ -11,26 +10,29 @@ app.get('/api/topics', getTopics);
 
 app.get('/api/articles', getArticles)
 
+app.get('/api/articles/:article_id', getArticlesById)
 
 app.use((err, request, response, next) => {
-    // console.log(err)
+    console.log(err)
     if (err.status){
-        console.log(err, ",___")
+        console.log(err)
```

Comment:
once you're happy that the error handling is set up as expected, can remove your floaty logs